### PR TITLE
docs: debugging

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This will add more clarification to the debugging section + fix some typos.
mentioned here: https://github.com/overhangio/tutor/issues/949

### CODE REVIEW

File: docs/dev.rst [hunk 1]
```diff
@@ -180,7 +183,7 @@ To check whether you have used the correct syntax, you should run ``tutor mounts
 
   $ tutor mounts add /path/to/edx-platform
   $ tutor mounts list
-  - name: /home/data/regis/projets/overhang/repos/edx/edx-platform
+  - name: /home/data/regis/projects/overhang/repos/edx/edx-platform
```
Comment (regisb) [inline docs/dev.rst:186]: This is actually my own home path :sweat_smile: please replace by `/path/to/edx-platform`.

**Review by regisb — APPROVED** [review]
Looking good! Just one minor comment. If necessary we can merge this as it is.

File: docs/dev.rst (hunk 1 above)
Comment (CodeWithEmad) [reply docs/dev.rst]: Sure 😅

**Review by regisb — APPROVED** [review]
Thanks!

**Pull request merged**